apps/qubit/modules/informationobject/templates/_nameAccessPoints.php

```diff
@@ -10,12 +10,14 @@
 
   <div>
     <ul>
-      <?php if (isset($sidebar)): ?>
+      <?php if (isset($showActorEvents) || isset($sidebar)): ?>
         <?php $actorsShown = array(); ?>
-        <?php foreach ($resource->getActorEvents() as $item): ?>
-          <?php if (isset($item->actor) && !isset($actorsShown[$item->actor->id])): ?>
-            <li><?php echo link_to(render_title($item->actor), array($item->actor)) ?> <span class="note2">(<?php echo render_value_inline($item->type->getRole()) ?>)</span></li>
-            <?php $actorsShown[$item->actor->id] = true; ?>
+	<?php foreach ($resource->getActorEvents() as $item): ?>
```

Comment:
Nitpick - clean up the indenting in this file so that the endifs and endforeach all line up and are indented correctly